ui/src/api/auth-api.ts

```diff
@@ -0,0 +1,11 @@
+import RestClient from './rest-client';
+
+export default class AuthApi {
+  static createUser(url: string, data: { email: string; password: string }) {
```

Comment:
Use an Interface for that.
In the future we will need a shared package to share DTOs across client/server apps